sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EQueryITest.scala

```diff
@@ -33,6 +33,10 @@ class SparkE2EQueryITest
   // to ensure we don't do sub-range feed-range
   // once emulator fixed switch back to default partitioning.
 
+  "spark items DataSource version" can "be determined" in {
+    CosmosItemsDataSource.version shouldEqual CosmosConstants.currentVersion
```

Comment:
Java SDK version doesn't say much - because we ship whatever currently is in Main - not the last released version. So just Spark version is sufficient - we need to infer form there what exactly the status of the Core SDK was.

This changes when we start only referencing the last released version if we ever do that.

This public API is mostly meant to help customers validate / check - we see both versions from the telemetry/user agent. And because they don't explicitly install Java SDK - just Spark connector - I think we don't need to add a public API for it.

That said - I personally would like to have a Java SDK version for similar reasons (unrelated to Spark) that is public.